great_expectations/expectations/metrics/query_metrics/query_table.py

```diff
@@ -22,7 +22,25 @@
 
 class QueryTable(QueryMetricProvider):
     metric_name = "query.table"
-    value_keys = ("query",)
+    value_keys = (
+        "query",
```

Comment:
Any reason not to remove query here?

Reply:
we currently utilize it in other expectations. I can update things across the board but that'll increase scope so I'll opt for a separate PR